### PR TITLE
Add tools overview page and IDX deprecation notice

### PR DESCRIPTION
**Pull request by PaulLeCam**
Also adds a link to the datamodels registry now that it's up.

### CODE REVIEW

File: docs/tools/idx/overview.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # IDX
 
+!!! warning "Deprecation notice"
+
+    :octicons-stop-16: The reference implementations for the Identity Index (IDX) specification are now provided by the [Glaze](../glaze/overview.md) and [Self.ID](../self-id/overview.md) projects.
```
Comment (oed) [inline docs/tools/idx/overview.md:5]: Should this refer to `did-datastore` specifically?

File: docs/tools/overview.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Self.ID is both a **reference application** and a **SDK** providing higher-level abstractions to create user-centric Web applications. It leverages Glaze projects and [3ID Connect](../authentication/3id-did/3id-connect.md) to offer a simple way to help Web developers get started with the Ceramic ecosystem.
+
+[Self.ID overview](self-id/overview.md){: .md-button }
+
+## IDX
+
+IDX refers to both the [Identity Index specification](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-11/CIP-11.md) and a set of tools (libraries and CLI).
+
+While the specification is still current, the IDX tools are now **deprecated** in favor of Glaze and Self.ID presented above.
```
Comment (oed) [inline docs/tools/overview.md:21]:
```suggestion
While the specification is still actively used, the IDX SDK and tools are now **deprecated** in favor of Glaze and Self.ID presented above.
```

**Review by oed — APPROVED** [review]
lgtm

File: docs/tools/idx/overview.md (hunk 1 above)
Comment (PaulLeCam) [reply docs/tools/idx/overview.md:5]: Specifically yes but I think it's more useful to go through the overview first, no?

Comment (oed) [reply docs/tools/idx/overview.md:5]: I mean just this sentence could say did-datastore, which is provided by glaze/self.id

Comment (PaulLeCam) [reply docs/tools/idx/overview.md]: OK I see, thanks! I updated the message, is it better now?

Comment (oed) [reply docs/tools/idx/overview.md]: yes, thanks!

**Pull request merged**